src/samtools/samtools_idxstats/test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+test_dir="${meta_resources_dir}test_data"
```

Comment:
meta_resources_dir does not always contain a trailing slash

```suggestion
test_dir="${meta_resources_dir}/test_data"
```